packages/react-ui/src/themes/dark.js

```diff
@@ -155,6 +155,8 @@ tokens.sizes = { ...tokens.space }
 */
 
 tokens.colors.text = {
+  textColor: 'grays.800',
+  themeColor: 'grays.800',
```

Comment:
what do these 2 tokens imply?

Reply:
themeColor token is to change the hover styling according to theme change i.e., dark and light.

textColor token is to make Element component text visible on the ./components page irrespective of theme change.

Reply:
Ah okay, I understand your solution, but don't think it is the right one.

The problem here is that the components page is using hard coded colors instead of using components or values from tokens

I've solved these in https://github.com/siddharthkp/react-ui/pull/74

Reply:
Okay, This looks great!!